js/compressed-token/src/program.ts

```
@@ -540,6 +639,57 @@
+     *
+     * @param poolPda The token pool pda to find the index and bump for
+     * @param mint The mint of the token pool
+     *
+     * @returns The index and bump number.
+     */
+    static findTokenPoolIndexAndBump(
+        poolPda: PublicKey,
+        mint: PublicKey,
+    ): [number, number] {
+        for (let index = 0; index < 5; index++) {
+            const derivedPda = this.deriveTokenPoolPdaWithIndex(mint, index);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)